src/rockstor/storageadmin/static/storageadmin/js/views/dashboard/pool_usage.js

```diff
@@ -3,7 +3,7 @@
  * @licstart  The following is the entire license notice for the
  * JavaScript code in this page.
  *
- * Copyright (c) 2016 RockStor, Inc. <http://rockstor.com>
+ * Copyright (c) 2013-2016 RockStor, Inc. <http://rockstor.com>
```

Comment:
Hi @gkadillak , not really sure about that

@schakrava ?!? :smiley: 

I think it can be good to have a "copyright var" to auto change years on builds